Week1/homework/connection.js

```diff
@@ -0,0 +1,101 @@
+  "INSERT INTO country VALUES ('Sweden', 'Europe', 'Nordic Countries', 449964, '836-01-01', 9910701, 79.6, NULL, NULL, NULL, 'Constitutional Monarchy', NULL, 'Stockholm')",
+  "INSERT INTO country VALUES ('Syria', 'Asia', 'Middle East', 185180, '1941-01-01', 18269868, 68.5, NULL, NULL, NULL, 'Republic', NULL, 'Damascus')",
+  "INSERT INTO country VALUES ('Tanzania', 'Africa', 'Eastern Africa', 883749, '1961-01-01', 57310019, 52.3, NULL, NULL, NULL, 'Republic', NULL, 'Dodoma')",
+];
+for (let i = 0; i < insert_country.length; i++) {
+  sendQuery(insert_country[i]);
+}
+
+//Create a table called 'city'.
+var create_city =
+  'create table city (ID int, Name varchar(50), Country_Code int, District varchar(50), Population int)';
+
+var insert_city = [
+  "INSERT INTO city VALUES (1, 'Amsterdam', 31, NULL, 821752)",
+  "INSERT INTO city VALUES (2, 'Rotterdam', 31, NULL, 623652)",
+  "INSERT INTO city VALUES (3, 'Kabul', 93, NULL, 4600000)",
+  "INSERT INTO city VALUES (4, 'Bucharest', 40, NULL, 1836000)",
+  "INSERT INTO city VALUES (5, 'Stockholm', 46, NULL, 965232)",
+  "INSERT INTO city VALUES (6, 'Damascus', 963, NULL, 1711000)",
+  "INSERT INTO city VALUES (7, 'Dodoma', 255, NULL, 2084000)",
+  "INSERT INTO city VALUES (8, 'Bern', 41, NULL, 133115)",
+];
+
+sendQuery(create_city);
+for (let i = 0; i < insert_country.length; i++) {
+  sendQuery(insert_city[i]);
+}
+//Write queries (that's a technical term meaning 'requests for information from a database') that will retrieve data that answers the following questions
+//Write queries for: What are the names of the countries with population greater than 8 million
+var select_by_population = 'SELECT name FROM country WHERE population > 8000000';
+sendQuery(select_by_population);
+
+//Write queries for: What are the names of the countries that have “land” in their names ?
+var select_by_land = "SELECT name FROM country WHERE name LIKE '%land%'";
+sendQuery(select_by_land);
+
+//Write queries for: What are the names of the cities with population in between 500,000 and 1 million ?
+var select_query = 'SELECT * FROM city WHERE Population BETWEEN 500000 AND 1000000';
+sendQuery(select_query);
+
+//Write queries for: What are the names of all the countries on the continent ‘Europe’ ?
+var select_by_continent = "SELECT name , continent FROM country WHERE continent = 'Europe'";
+sendQuery(select_by_continent);
+
+//Write queries for: List all the countries in the descending order based on their surface areas.
+var select_by_surfaceArea =
+  'SELECT name AS Country_Name, Surface_Area FROM country ORDER BY surface_Area DESC';
+sendQuery(select_by_surfaceArea);
+
+//Bonus Write queries that answer the following questions:
+
+//What are the names of all the cities in the Netherlands?
+var select_by_citiesOfNetherlands = 'SELECT name AS City_Name FROM city WHERE Country_Code = 31';
```

Comment:
are you familiar with the term magic number? https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

This can be written much more clearly:
```
const NETHERLANDS_CODE = 31;
var select_by_citiesOfNetherlands = 'SELECT name AS City_Name FROM city WHERE Country_Code = ${NETHERLANDS_CODE};
```

Reply:
Thank you for this information, I didn't know the term. I appreciate it very much